### ISSUE TITLE
Error creating global storage with only default tenant

### ISSUE THREAD

**Issue by SteveH-US** [starkandwayne/shield#484]
I deployed a brand new Shield v8.0.17 instance, using the "shield-boshdeployment" v8.0.18, all on one VM. 

The "tenant1" tenant existed, but with no members, I added the Administrator to the tenant as with the "Administrator" role. 

When I attempt to create global storage, I received the following error:
```
Uncaught TypeError: Cannot read property 'uuid' of undefined
    at HTMLSelectElement.<anonymous> (shield.js:1233)
    at HTMLFormElement.dispatch (shield.js:3)
    at HTMLFormElement.q.handle (shield.js:3)
```

I was able to get past this, but creating a new tenant, again adding the Administrator users (as Administrator).

**Comment by jhunt** [comment]
This should be fixed in the 8.1.0 release, which should be out the door by next week (just in time for CF Summit Philly!)

**Comment by jhunt** [comment]
This is fixed in v8.1.0 (https://github.com/starkandwayne/shield/releases/tag/v8.1.0)
